static/sass/_pattern_cube-progression.scss

```diff
@@ -0,0 +1,201 @@
+@mixin ubuntu-p-cube-progression {
+  $diamondSize: (
+    "width": 168,
+    "height": 99,
+  );
+  $rhombusSize: (
+    "width": 85,
+    "height": 145,
+  );
+
+  .p-cube-progression {
+    height: 382px;
+    position: relative;
+
+    .is-faded {
+      background-color: rgba(255, 255, 255, 0.3);
+      clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+      opacity: 0.5;
+
+      &--right {
+        @extend .is-faded;
+
+        clip-path: polygon(0% 34%, 100% 0%, 100% 66%, 0% 100%);
+      }
+
+      &--left {
+        @extend .is-faded;
+
+        clip-path: polygon(0% 0%, 100% 34%, 100% 100%, 0% 66%);
+      }
+    }
+
+    %diamond-badge {
+      background-size: map-get($diamondSize, "width") * 1px
+        map-get($diamondSize, "height") * 1px;
+      height: map-get($diamondSize, "height") * 1px;
+      position: absolute;
+      width: map-get($diamondSize, "width") * 1px;
+    }
+
+    %rhombus-badge {
+      background-size: map-get($rhombusSize, "width") * 1px
+        map-get($rhombusSize, "height") * 1px;
+      height: map-get($rhombusSize, "height") * 1px;
+      position: absolute;
+      width: map-get($rhombusSize, "width") * 1px;
+    }
+
+    &__tag {
```

Comment:
I think theses should be the full class names rather than concatenated. We've agreed in dev catchup a few times that is the right approach, so in this case it would be `.p-cube-progression__tag`.

Concatenating class names in this way makes them impossible to search for in the codebase and on a file that is taller than your screen they are very hard to follow. It's better to be explicit.

Reply:
Thanks for your help @steverydz, makes sense I'll change it now 👍🏼